map/src/data/DOCUMENTATION.md

```diff
@@ -0,0 +1,110 @@
+# Map Marker Documentation
```

Comment:
I would call this file `README.md` just so it appears in the directory view of `map/src/data` :)